package.json

```diff
@@ -26,23 +28,24 @@
         "jgs:start": "BROWSER=none yarn start --proxy https://debug.dhis2.org/dev"
     },
     "devDependencies": {
+        "@cypress/webpack-preprocessor": "^5.17.0",
```

Comment:
Shouldn't this be provided by the cypress-plugins package?

Reply:
Well, it's only required when using cucumber, it's a peer dependency of the cucumber plugin. The cypress-cucumber-preprocessor is marked as a peer dependency as well as the app needs to import and use it directly. So I thought they both should be peer dependencies. Would you then move both deps to the cypress plugins lib?

Reply:
Ah I see, I didn't know they were peerDeps of `@dhis2/cypress-plugins`. In that case this makes sense yeah. One thing you could do in the `@dhis2/cypress-plugins` package is mark any peerdependencies that aren't always necessary as optional: https://docs.npmjs.com/cli/v9/configuring-npm/package-json#peerdependenciesmeta

> When a user installs your package, npm will emit warnings if packages specified in peerDependencies are not already installed. The peerDependenciesMeta field serves to provide npm more information on how your peer dependencies are to be used. Specifically, it allows peer dependencies to be marked as optional.

Depending on whether you want users to be able to omit installation of `cypress-cucumber-preprocessor` and the webpack processor if they're not using cucumber.